### PR TITLE
fix:hotfixes

### PR DESCRIPTION
**Pull request by mohsenParvar**

### CODE REVIEW

File: components/atoms/details2/components/header.js [hunk 1]
```diff
@@ -31,15 +33,16 @@ export const Details2Header = ({
           />
         )}
 
-        <Text size={SIZE.MD} context='dark' content={title} />
+        <Text size={SIZE.MD} context={titleContext || 'dark'} content={title} />
```
Comment (majid-amiri) [inline components/atoms/details2/components/header.js:36]: Use defaultProps instead of `|| dark`

File: components/atoms/text/text.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ import styled, { css } from 'styled-components'
 
 export const Text = ({
   align,
+  bold,
```
Comment (majid-amiri) [inline components/atoms/text/text.js:14]: Isn't it better to add a prop like `weight` and have values like `light, bold, semi-bold, ...` there based on our designs?

File: components/organisms/login/login.js [hunk 3]
```diff
@@ -34,6 +34,7 @@ export const Login = ({
   blockSubmitButton,
   forgotPassword,
   heading,
+  noTitle,
```
Comment (majid-amiri) [inline components/organisms/login/login.js:37]: Change this to `showTitle` with a `true` defaultValue. The condition should also be changed to `{showTitle && </>} `

File: components/atoms/text/text.js [hunk 4]
```diff
@@ -11,23 +11,27 @@ import styled, { css } from 'styled-components'
...
```
Comment (majid-amiri) [inline components/atoms/text/text.js:14]: Please remove `bold` now that you added `weight`

File: components/atoms/details2/components/header.js (hunk 1 above)
Comment (majid-amiri) [reply components/atoms/details2/components/header.js]: Did you fix this? @mohsenParvar

**Review by majid-amiri — CHANGES_REQUESTED** [review]
Please fix the stuff I mentioned in the comments - thanks

File: components/atoms/details2/components/header.js (hunk 1 above)
Comment (mohsenParvar) [reply components/atoms/details2/components/header.js]: @majid-amiri it is fixed

File: components/atoms/text/text.js (hunk 4 above)
Comment (mohsenParvar) [reply components/atoms/text/text.js:14]: @majid-amiri i think having "bold" as props is handy, also it is used in iui website and maitai,
having <Text bold /> i think looks better than <Text weight='semiBold' />

Comment (majid-amiri) [reply components/atoms/text/text.js:14]: @mohsenParvar It doesn't make sense to me to have both `bold` and `weight`. Please remove `bold` and change the usages on other products as well.

File: components/atoms/details2/components/header.js (hunk 1 above)
Comment (majid-amiri) [reply components/atoms/details2/components/header.js]: @mohsenParvar Why I can't see the change? I still see `context={titleContext || 'dark'}`

**Review by majid-amiri — APPROVED** [review]

**Pull request merged**